phittipaldi/openbudget
# add github social account to login

## Status
:white_check_mark: READY/ :wrench: IN DEVELOPMENT

## Migrations
YES

## Description
This feature help the not authenticated user to login with a github account. close #7 

## Deploy Notes
Install django-allauth==0.34.0 and make the corresponding migrate for this app. This will create the tables to manage social accounts.

## Steps to Test or Reproduce
1. Login to admin site
2. Create social application with this data: **Client id:** a7782410b239caccaf30 **Secret Key:** 882d8937248cec3448244a80b65102791872f5da
3. Make sure your setting has all the configuration corresponding to django-allauth app.
4. Finally Join this feature clicking in GitHub Button in the Login Page.

## Impacted Areas in Application
Security App
Setting, Main Urls
Loggin Page


Please, before doing the PR make sure that you have completed the following checklist

**Checklist**
- [x] Docstrings in my code follow the convention for this project.

Thanks for contributing to Open Budget
